### ISSUE TITLE
Wrong status-go version displayed in About screen

### ISSUE THREAD

**Issue by igor-sirotin** [status-im/status-desktop#16075]
# Bug Report

## Description

<img width="398" alt="image" src="https://github.com/user-attachments/assets/7892db06-d614-405d-a726-6863517a620b">

```nim
INFO [08-12|10:22:16.041|github.com/status-im/status-go/api/geth_backend.go:2076] 
status-go version details                
package=status-go/api.GethStatusBackend 
version= 
commit=a827dee54
```

**Comment by igor-sirotin** [comment]
Related, but a different issue: https://github.com/status-im/status-desktop/issues/12349

**Comment by caybro** [comment]
> Related, but a different issue: #12349

Both arriving at compile time; from `constants.nim`:
```nim
# This is changed during compilation by reading the VERSION file
const DESKTOP_VERSION {.strdefine.} = "0.0.0"
const STATUSGO_VERSION* {.strdefine.} = "0.0.0"
```

**Comment by caybro** [comment]
And then from `Makefile`:
```
# App version
VERSIONFILE=VERSION
SGVERSIONFILE=vendor/status-go/VERSION
DESKTOP_VERSION=`cat $(VERSIONFILE)`
STATUSGO_VERSION=`cat $(SGVERSIONFILE)`
```

**Comment by caybro** [comment]
TLDR, `vendor/status-go/VERSION` is incorrect

**Comment by caybro** [comment]
```
❯ cat VERSION 
0.179.11
```

There is a more up to date tag but the VERSION file didn't get the bump:
![image](https://github.com/user-attachments/assets/0459d5fc-f61a-4416-9014-4f61a085fa7f)

**Comment by igor-sirotin** [comment]
> TLDR, `vendor/status-go/VERSION` is incorrect

Yes. But it's not used anymore at all. 
The version is stored as git tag nowdays.

**Comment by caybro** [comment]
Manage to fix it on our side:

![image](https://github.com/user-attachments/assets/69e3a327-b18c-42eb-945e-150670235a78)

But the status-go code still outputs the wrong version at startup:
```
INFO [08-12|17:41:21.400] Status backend initialized               backend=geth version=0.179.11 commit=48ebe24f8 IpfsGatewayURL=https://ipfs.status.im/
```

Looks like the `VERSION` file is still used in the code itself @igor-sirotin ?

**Comment by caybro** [comment]
(disregard the `Qt version` there :wink: ) This is something I wanted to add too anyways
